lib/src/generated/platformconstants.dart

```diff
@@ -161,3 +167,21 @@ class TxMessage {
     static const String name = "name";
     static const String extras = "extras";
 }
+
+class TxPaginatedResult {
```

Comment:
[avoid_classes_with_only_static_members ](https://dart-lang.github.io/linter/lints/avoid_classes_with_only_static_members.html) 🤔

Reply:
We disabled this lint rule.
https://github.com/ably/ably-flutter/blob/feature/lint-dart/analysis_options.yaml#L77

Reply:
ok, fair enough. Thanks. 😁